webapp/src/components/modals/deleteModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+
+import './styles.scss';
+
+type Props = {
+    visibility: boolean;
+    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
+    config: Configs[];
+    configIndex: number;
+    onChange: (config: Configs[]) => void;
```

Comment:
give a more descriptive name here also

Reply:
please elaborate

Reply:
Like we are doing in config modal. something like "onConfigChange"